aana/tests/deployments/test_vad_deployment.py

```diff
@@ -1,100 +1,60 @@
 # ruff: noqa: S101
-import json
 from importlib import resources
 from pathlib import Path
 
 import pytest
-from ray import serve
 
 from aana.core.models.audio import Audio
 from aana.core.models.base import pydantic_to_dict
 from aana.core.models.vad import VadParams
-from aana.tests.utils import (
-    get_deployments_by_type,
-    is_gpu_available,
-    is_using_deployment_cache,
-)
-
-
-def compare_vad_outputs(expected_output, predictions):
-    """Compare two vad outputs.
-
-    Start and end positions are compared using Levenshtein distance.
-
-    Args:
-        expected_output (dict(list[dict])): the expected vad_output
-        predictions (dict(list[dict])): the predictions of vad
-
-    Raises:
-        AssertionError: if vad_outputs differ too much
-    """
-    # Number of 30 sec segments should not change.
-    assert len(expected_output["segments"]) == len(predictions["segments"])
-
-    if len(expected_output["segments"]) != 0:  # non-empty files
-        # all start and end times should be similar and number of segments should be same.
-        for expected, predicted in zip(
-            expected_output["segments"], predictions["segments"], strict=False
-        ):
-            assert (
-                abs(
-                    expected["time_interval"]["start"]
-                    - predicted["time_interval"]["start"]
-                )
-                < 2.0
-            )  # all segment starts within 2 sec
-            assert (
-                abs(
-                    expected["time_interval"]["end"] - predicted["time_interval"]["end"]
-                )
-                < 2.0
-            )  # all segment ends within 2 sec
-
-            # check same number of small voiced segments within each vad segment
-            assert len(expected["segments"]) == len(predicted["segments"])
-
+from aana.deployments.aana_deployment_handle import AanaDeploymentHandle
+from aana.deployments.vad_deployment import VadConfig, VadDeployment
+from aana.tests.utils import verify_deployment_results
+
+deployments = [
+    (
+        "vad_deployment",
+        VadDeployment.options(
+            num_replicas=1,
+            max_ongoing_requests=1000,
+            ray_actor_options={"num_gpus": 0.05},
+            user_config=VadConfig(
+                model=(
+                    "https://whisperx.s3.eu-west-2.amazonaws.com/model_weights/segmentation/"
```

Comment:
Maybe not for now, but we will be on the hook for S3 egress every time someone runs this test. We should think of a different solution at some point.

Reply:
This isn't our S3 bucket. Someone else is on the hook 😃 But in general, the idea is to switch to the HF hub and use it for all models. We even have an issue for that: https://github.com/mobiusml/aana_sdk/issues/65